docs/examples/custom/deployment-script-with-storage/main.bicep

```diff
@@ -20,7 +20,7 @@ resource stg 'Microsoft.Storage/storageAccounts@2019-06-01' = {
   }
 }
 
-resource dScript 'Microsoft.Resources/deploymentScripts@2019-10-01-preview' = {
+resource dScript 'Microsoft.Resources/deploymentScripts@2020-10-01' = {

```

Comment:
Is that API version live?

Reply:
It better be! Swagger has been published: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/resources/resource-manager/Microsoft.Resources/stable/2020-10-01/deploymentScripts.json